roles/third_party/ldap-install/templates/users.ldif.j2

```diff
@@ -4,14 +4,14 @@ objectClass: top
 dc: {{ __ldap_tld }}
 
 
-dn: uid={{ __ldap_userid }}1,{{ __ldap_olc_suffix }}
+dn: uid={{ __ldap_admin_user }}1,{{ __ldap_olc_suffix }}
```

Comment:
I'm probably missing the context here, but __ldap_admin_user is, by default, Admin, the user which has in context admin rights on LDAP. 

__ldap_userid, in another hand, is some user ID you can give it to create a bunch of users where the first of them will become CNX admin as well, so those are two different domains. I'm not sure I would change this.